.github/workflows/testnet-release.yml

```diff
@@ -0,0 +1,32 @@
+name: "Testnet Community Release"
+
+on:
+  push: 
+    branches: 
+      - main
+env:
+  CHAIN: "picasso"
+  RELEASE_VERSION: "latest"
+
+jobs:
+  build-and-publish:
+    runs-on: 
+        - self-hosted
+        - linux
+        - x64
+        - sre
+    steps:
+    - uses: actions/checkout@v2
+      with: 
+        fetch-depth: 0
+        ref: picasso
+
+    - name: Build and Push Artifacts to gcloud
+      run: |
+        /home/runner/.cargo/bin/cargo build --release --bins
+        git rev-parse HEAD > revision
+        tar -czvf composable-${{ env.RELEASE_VERSION }}.tar.gz target/release/composable
```

Comment:
Is `RELEASE_VERSION` existing here? Isn't this supposed to be set from the `revision` file?
Also I think devops used the scheme `composable-picasso-{version}` when publishing

Reply:
> Is RELEASE_VERSION existing here?

Yes, `RELEASE_VERSION: "latest"`.

> Also I think devops used the scheme composable-picasso-{version} when publishing

Fixed.

> Isn't this supposed to be set from the revision file?

A static URL needs here (e.g. `https://storage.googleapis.com/composable-binaries/testnet-releases/dali/composable-dali-latest.tar.gz`) to use it in https://github.com/ComposableFi/composable/blob/main/.maintain/playbooks/run-integration-tests.yml#L131. Also, we quick full bucket of storage if the version will equal revision.